test/bench/tree_editing_bench_test.go

```diff
@@ -0,0 +1,75 @@
+//go:build bench
+
+/*
+ * Copyright 2024 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bench
+
+import (
+	"testing"
+
+	"github.com/yorkie-team/yorkie/api/converter"
+	api "github.com/yorkie-team/yorkie/api/yorkie/v1"
+	"github.com/yorkie-team/yorkie/pkg/document/json"
+	"github.com/yorkie-team/yorkie/test/helper"
+)
+
+func BenchmarkTreeEditing(b *testing.B) {
+	BenchmarkTreeConverting(b)
+}
+
```

Comment:
Go's benchmark automatically recognizes and runs functions that start with `BenchmarkXXX`. Therefore, there is no need for `BenchmarkTreeEditing` to call `BenchmarkTreeConverting` again.

<img width="1148" alt="Screenshot 2024-07-24 at 9 37 10 AM" src="https://github.com/user-attachments/assets/1c44aa95-6135-4121-93f1-1276ebcdf797">

```suggestion
```